userprofile/views.py

```diff
@@ -170,6 +170,33 @@ def get_common_objects(self):
         self.accessibility_statement = accessibility_statement
         self.note("accessibility_statement")
 
+class SupportView(UserProfileView):
+    access_mode=ACCESS.ANONYMOUS
+    template_name="userprofile/support.html"
+    extra_context = {
+        'brand_name': settings_text('BRAND_NAME')
+    }
+
+    def get_common_objects(self):
+        super().get_common_objects()
+        lang = "_" + get_language().lower()
+        key = make_template_fragment_key('support_channels', [lang])
+        support_channels = cache.get(key)
+        if not support_channels:
+            template_name = "support_channels{}.html"
+            template = try_get_template(template_name.format(lang))
+            if not template and len(lang) > 3:
+                template = try_get_template(template_name.format(lang[:3]))
+            if not template:
+                logger.warning("No localized support for language %s", lang)
+                template = try_get_template(template_name.format(''))
+            if not template:
+                logger.error("The support page is missing")
+
+            support_channels = template.render() if template else _("Please send an email to ") + "<a href='mailto:aplusguru@cs.aalto.fi'>aplusguru@cs.aalto.fi</a>"
```

Comment:
We shouldn't hardcode our aplusguru email address in the source code. If the site has no local template, can't we just render the default base support template here?

Reply:
Do you mean this?
![image](https://user-images.githubusercontent.com/10437475/105471171-1e238100-5ca3-11eb-866e-96f6ec1ff36f.png)

Reply:
Now I don't understand. But you include the default template in this PR. Is it not always possible to render that template even if the site has not installed any `local_templates` of their own?

Reply:
Sorry, I was thinking about something different. I will implement this. I will take the accessibility statement code as a guideline